### PR TITLE
New functionality about java jokes added

### PR DESCRIPTION
**Pull request by Edu7216**
If the actual day is even, one joke is shown. Other if the actual day is odd.

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fjlopez) [inline .classpath]: Remove this file and add check why this file has not been ignored in your environment as it is in `.gitignore`.

File: .project [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fjlopez) [inline .project]: Remove this file and add check why this file has not been ignored in your environment as it is in `.gitignore`.

File: .settings/org.eclipse.buildship.core.prefs [hunk 3]
```diff
@@ -0,0 +1,2 @@
+connection.project.dir=
```
Comment (fjlopez) [inline .settings/org.eclipse.buildship.core.prefs]: Remove this file and add check why the folder `.settings` has not been ignored in your environment as it is in `.gitignore`.

File: .vscode/launch.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (fjlopez) [inline .vscode/launch.json]: Remove this file and add check why the folder `.vscode` has not been ignored in your environment as it is in `.gitignore`.

File: bin/main/application.properties [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# View Resolver
```
Comment (fjlopez) [inline bin/main/application.properties]: Remove this file and add check why the folder `bin` has not been ignored in your environment as it is in `.gitignore`.

File: src/main/webapp/WEB-INF/jsp/joke.jsp [hunk 6]
```diff
@@ -0,0 +1,27 @@
+<%-- IntelliJ specific linter annotations --%>
```
Comment (fjlopez) [inline src/main/webapp/WEB-INF/jsp/joke.jsp]: JSP code starts to repeat (see `wellcome.jsp`). If you want a :gift: discover how to factor out the repeated code in both files in a third jsp (or other technology) so each jsp can import the shared layout.

**Comment by fjlopez** [comment]
Ah, and do not forget that you should also resolve the conflict with: 

```
README.md
src/main/java/es/unizar/webeng/hello/HelloController.java
```

**Comment by fjlopez** [comment]
@Edu7216 solve these issues ASAP.

File: bin/main/application.properties (hunk 5 above)
Comment (Edu7216) [reply bin/main/application.properties]: File removed

File: .vscode/launch.json (hunk 4 above)
Comment (Edu7216) [reply .vscode/launch.json]: File removed

File: .settings/org.eclipse.buildship.core.prefs (hunk 3 above)
Comment (Edu7216) [reply .settings/org.eclipse.buildship.core.prefs]: File removed

File: .project (hunk 2 above)
Comment (Edu7216) [reply .project]: File removed

File: .classpath (hunk 1 above)
Comment (Edu7216) [reply .classpath]: File removed

**Comment by fjlopez** [comment]
Remember to remove the files from the `bin` folder too.

**Pull request merged**